projects/demo/src/modules/experimental/pin/pin.template.html

```diff
@@ -0,0 +1,96 @@
+<tui-doc-page
+    header="Pin"
+    package="EXPERIMENTAL"
+    type="components"
+>
+    <ng-template pageTab>
+        <tui-notification status="warning">
+            This code is
+            <strong>experimental</strong>
+            and is a subject to change. Expect final solution to be shipped in the next major version
+        </tui-notification>
+
+        <tui-doc-example
+            id="default"
+            heading="Default"
+            [content]="example1"
+            [fullsize]="true"
+        >
+            <tui-pin-example-1></tui-pin-example-1>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="dot"
+            heading="Dot"
+            [content]="example2"
+            [fullsize]="true"
+        >
+            <tui-pin-example-2></tui-pin-example-2>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="selected"
+            heading="Selected"
+            [content]="example3"
+            [fullsize]="true"
+        >
+            <tui-pin-example-3></tui-pin-example-3>
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <div class="preview">
+                <div [(tuiPin)]="open">
+                    <tui-svg [src]="icon"></tui-svg>
+                </div>
+            </div>
+        </tui-doc-demo>
+
+        <tui-doc-documentation>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="src"
```

Comment:
You ended up removing this as input. I guess the whole API tab is rather unnecessary in this case, it's just one input and all other cases are covered by examples.

Reply:
fixed